modules/reference/pages/feature/socialLogin/examples.adoc

```diff
@@ -64,16 +65,16 @@ The user must authenticate through Google and also verify as an existing user in
 
 ----
 
-For more information on configuring an LDAP registry, see the LDAP User Registry (feature:ldapRegistry[]) feature.
+For more information on configuring an LDAP registry, see the feature:ldapRegistry[display=LDAP User Registry feature].
 
 === Request log in with a social media ID or with their account for the configured registry
 
 You can give users the option of logging in with either a social media provider or with their account on the configured registry.
-For example, use the `enableLocalAuthentication` attribute to configure your app so that users can have the option of logging in with a Google ID or with their account on their company's LDAP registry:
+For example, use the `enableLocalAuthentication` attribute to configure your application so that users can log in with a Google ID or with their account on their company's LDAP registry:
 
 [source,xml]
 ----
-<!-- user will be presented choice menu of either Google or ldap -->
+<!-- user will be presented a choice menu of either Google or LDAP -->
```

Comment:
missed this the first time- avoid future tense

user will be presented a choice menu of either Google or LDAP
--->
The user is given a choice menu of either Google or LDAP

Reply:
Correction made: The user will be given a choice menu of either Google or LDAP